src/utils/CaptainInstaller.ts

```diff
@@ -301,7 +301,10 @@ export function install() {
         .then(function () {
             const imageName = CaptainConstants.configs.certbotImageName
             console.log(`Pulling: ${imageName}`)
-            return DockerApi.get().pullImage(imageName, undefined)
+            return DockerApi.get().pullImage(imageName, undefined).catch(function (onrejected) {
+                console.log(`Continnuing failed pull, maybe image already exists: ${imageName}.`, onrejected)
+                return Promise.resolve()
```

Comment:
This check was added to early exit in cases where images cannot be fetched from Dockerhub due to firewall restrictions. I understand it was needed for debugging, but let's revert it for prod. 

Maybe as a separate change, we can actually check if the image exists already and only then let go of the error.

Reply:
Reverted.